packages/app-layout/AppLayout.tsx

```diff
@@ -0,0 +1,161 @@
+import { h, Component, prop } from 'skatejs';
+
+import {
+  GenericTypes,
+  matchMedia
+} from '../_helpers';
+
+import { HamburgerButton } from './components/HamburgerButton';
+import { Drawer } from './components/Drawer';
+import { Overlay } from './components/Overlay';
+import { Nav } from './components/Nav';
+import { NavContent } from './components/NavContent';
+
+import styles from './AppLayout.scss';
+
+const DEFAULT_RESPONSIBLE_WIDTH = '640px';
```

Comment:
Just to think about:
- is it really good to have user (when specifying `responsiveWidth`) use string with some units?
- wouldn't it be better to have the width as a number and just always use **px**?